src/ImathTest/CMakeLists.txt

```diff
@@ -84,6 +84,10 @@ function(DEFINE_IMATH_TESTS)
   endforeach()
 endfunction()
 
+if (TBB_BUILD_APPLE_FRAMEWORKS)
```

Comment:
did you mean to write `IMATH_BUILD_APPLE_FRAMEWORKS` here? Imath doesn't use TBB.